### DISCUSSION TITLE
Include concept number in video names

### DISCUSSION THREAD

**Discussion by t-vi** [OpenMined/courses#75] [Feedback] [answered]
Thank you for making the course available!
I'm following the course in a semi-offline way. One very minor thing that would make this easier is if the concept numbering would be included in the video titles so the videos sort themselves.

Best regards

Thomas

**Comment by yemikifouly** [comment]
Hi Thomas! 👋 That is a good point. Adding it to the list of feature request to pass to our Web Developer. Thanks for bringing it up! 🙂

**Comment by mcleonard** [comment, marked as answer]
Tracking here: https://github.com/OpenMined/courses/issues/100
